Vagrantfile

```diff
@@ -11,6 +11,8 @@ Vagrant.configure("2") do |config|
   config.vm.box = 'dummy'
   config.vm.box_url = 'https://github.com/mitchellh/vagrant-aws/raw/master/dummy.box'
 
+  `find -L . -type l -delete`
```

Comment:
This should *definitely* have some commentary around it, why are you deleting all my sym links!